great_expectations/compatibility/sqlalchemy_and_pandas.py

```diff
@@ -45,3 +45,84 @@ def execute_pandas_reader_fn(
     else:
         reader_fn_result = reader_fn(**reader_options)
     return reader_fn_result
+
+
+def execute_pandas_to_datetime(
+    arg,
+    errors: str = "raise",
```

Comment:
non-blocking

Could use a `typing_extensions.Literal` here, to help narrow the possible values here. I don't think it's that useful here since this is mostly just a passthrough but wanted to bring it up regardless.
```suggestion
    errors: Literal["raise", "coerce", "ignore"] = "raise",
```

Reply:
updating now :)